lib/command.c

```diff
@@ -715,6 +715,46 @@ cmd_variable_complete (struct cmd_token *token, const char *arg, vector comps)
   vector_free(tmpcomps);
 }
 
+char *
+cmd_variable_comp2str(vector comps, unsigned short ind, unsigned short cols,
+                      const char nl[])
+{
+   size_t bsz = 8;
+   char *buf = XCALLOC(MTYPE_TMP, bsz);
+
+   char indent[ind + 1];
+   memset(indent, ' ', sizeof(indent) - 1);
+   indent[ind] = 0x00;
+
+   snprintf(buf, bsz, "%s", indent);
```

Comment:
`snprintf(buf, bsz, "%*s", ind, "");`

Also, I would suggest making `ind` a `#define AUTOCOMP_INDENT 5` and ditching the function parameter.